slides/03/03.md

```diff
@@ -569,7 +569,7 @@ In cross-validation, we choose multiple validation sets from the training data,
 and for each one, we train a model on the rest of the training data and
 evaluate on the chosen validation sets. A commonly used strategy to choose
 the validation sets is called **k-fold cross-validation**. Here the training set is partitioned
-into $k$ subsets of approximately the same size, and each subset takes turn
+into $k$ subsets of approximately the same size, and each subset takes a turn
```

Comment:
When doing some reading, it seems the correct phrase is "take turns" https://www.merriam-webster.com/dictionary/take%20turns; "take a turn" seems to be something different, at least according to https://www.merriam-webster.com/dictionary/take%20a%20turn. I will update it after merging.